libqfieldsync/offline_converter.py

```diff
@@ -135,7 +138,9 @@ def convert(self, reload_original_project: bool = True) -> None:
         """
         project = QgsProject.instance()
         self.original_filename = Path(project.fileName())
-        self.backup_filename = make_temp_qgis_file(project)
+        self.backup_filename = make_temp_qgis_file(
+            project, self._export_filename, self._export_title
+        )
```

Comment:
Why you need this? Isn't the export actually done as part of line 280 in the `export_project_filename` variable?

Reply:
Now I notice!.... I will create the respective changes!